packages/block-library/src/navigation/constants.js

```diff
@@ -14,3 +14,9 @@ export const ALLOWED_BLOCKS = [
 	'core/navigation-submenu',
 	'core/loginout',
 ];
+
+export const INSERTER_PRIORITY = [
+	'core/spacer',
```

Comment:
did we mean to put the spacer here or is this for testing?

Reply:
Nice catch 😓